.gitignore

```diff
@@ -31,7 +31,7 @@ __pycache__/
 .Python
 build/
 develop-eggs/
-dist/
+# dist/
```

Comment:
Please undo, we aren't including binaries in the repo.

Reply:
In addition to what @ptwng said, please also remove the `dist` directory from the repo and then rebase the commit to remove all commits where it was present.